src/Components/Gene/NewContents.tsx

```diff
@@ -36,7 +36,7 @@ class GeneNewContents extends React.Component<Props, State> {
 
   onDropdownSelect(slice: string, value: string) {
     this.setState({
-      [slice.toLowerCase()]: value,
+      [slice.toLowerCase() as any]: value,
```

Comment:
lol

Reply:
Yep 😝 

In short, `Pick` currently only works when literals are used as keys, or of course ‘anything’. See the linked SO Q/A and the TS issue the answer links to https://stackoverflow.com/questions/42090191/picks-k-type-with-dynamic-computed-keys/42131589#42131589

Reply:
Wow, super informative, thanks!